client/src/containers/projects/custom-project/header/parameters/index.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  COST_TYPE_SELECTOR,
+  PROJECT_PRICE_TYPE,
+} from "@shared/entities/projects.entity";
+import { z } from "zod";
+
+import { FILTER_KEYS } from "@/app/(overview)/constants";
+import { filtersSchema } from "@/app/(overview)/url-store";
```

Comment:
You have your own `filtersSchema` in the container, but you're still referring to the one of the other page. This is confusing.